docs/policy-only.md

```diff
@@ -0,0 +1,170 @@
+# Running Antrea In Policy Only Mode
+
+Antrea supports chaining with routed CNI implementations such as EKS CNI. In this mode, Antrea
+enforces Kubernetes NetworkPolicy, and delegates Pod IP management and network connectivity to the
+primary CNI.
+
+## Design
+
+Antrea is designed to work as NetworkPolicy plug-in for the routed CNIs. For as long as a CNI
```

Comment:
Question - does policy-only mode work only for routed CNI?

Consider: Antrea is designed to work as the NetworkPolicy plugin together with a routed CNI plugin.

Reply:
Guess remove "For" before "as long as".

Reply:
done.